charts/aws-fluent-bit/values.yaml

```diff
@@ -132,10 +140,14 @@ daemonSetVolumes:
 
 daemonSetVolumeMounts:
   - name: varlog
-    mountPath: /var/log
+    mountPath: /var/log/
+    readOnly: true
   - name: varlibdockercontainers
     mountPath: /var/lib/docker/containers
     readOnly: true
   - name: etcmachineid
     mountPath: /etc/machine-id
     readOnly: true
+
+flush: 1
```

Comment:
I presume this is config for Fluent Bit?

Reply:
yes